LibtorrentRasterbarConfig.cmake.in

```diff
@@ -1,16 +1,9 @@
 # - Config file for the @PROJECT_NAME@ package
-# It defines the following variables
-#  @PROJECT_NAME@_INCLUDE_DIRS - include directories for @PROJECT_NAME@
-#  @PROJECT_NAME@_LIBRARIES    - libraries to link against
+# It defines the LibtorrentRasterbar::torrent-rasterbar target to link against
 
 @PACKAGE_INIT@
 
 include(CMakeFindDependencyMacro)
 @_find_dependency_calls@
 
 include("${CMAKE_CURRENT_LIST_DIR}/LibtorrentRasterbarTargets.cmake")
-
-get_target_property(@PROJECT_NAME@_INCLUDE_DIRS LibtorrentRasterbar::torrent-rasterbar INTERFACE_INCLUDE_DIRECTORIES)
```

Comment:
isn't there a risk that someone depends on this, which would break in the next libtorrent release?

Reply:
Yes, of course. But I doubt people could prefer raw variables over the complete target.